common.ts

```diff
@@ -268,17 +268,16 @@ export class CodecVisitor<R> {
   }
 
   /**
-   * Once Deno releases with TS 4.7, this can be used like so:
    * ```ts
-   * visitor.generic((visitor) => <T>() =>
+   * visitor.generic(<T>() =>
```

Comment:
This is a bit cleaner IMO; didn't really like putting the old version into the docs